encrypted-media-respec.html

```diff
@@ -3108,19 +3108,63 @@
       <section id="persistent-state-requirements">
         <h3>Persistent Data</h3>
         <p>
-          Persistent data includes all data stored by the CDM, or by the User Agent on behalf of the CDM, that exists after the destruction of the <a>MediaKeys</a> object. Specifically, it includes any identifiers, licenses, keys, key IDs, key usage information or records of license destruction stored by the CDM or by the User Agent on behalf of the CDM.
+          Persistent Data includes all data stored by the CDM, or by the User Agent on behalf of the CDM, that exists after the destruction of the <a>MediaKeys</a> object. Specifically, it includes any identifiers, licenses, keys, key IDs, key usage information or records of license destruction stored by the CDM or by the User Agent on behalf of the CDM.
```

Comment:
11.6 Incomplete Clearing of Data specifically mentions Distinctive Identifiers and included it in the mitigations moved here. Since one might not think of a Distinctive Identifier as "Persistent Data," I think we should specifically mention it here.